pagopa/io-app
# feat: [IOPID-92] Add logic to use test login with lollipop

>[!Warning]
> This PR is related to this PR on the io-dev-api-server: https://github.com/pagopa/io-dev-api-server/pull/400

[![Run e2e tests](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml/badge.svg?branch=IOPID-92-align-test-account-to-lollipop)](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml)


## List of changes proposed in this pull request
- Add the logic to use lollipop with test login
- replace deprecated component 

## DEMO
<p>

| PRODUCTION TEST | DEV SERVER TEST |
| - | - |
| <video src="https://github.com/user-attachments/assets/d18b5b3c-055d-4bec-9392-9299bfd3bec0"/> | <video src="https://github.com/user-attachments/assets/59654588-f0da-4c78-8e92-57de6f352bc4"/> |

</p>

## How to test
- Start the app (using .env.production or .env.local environment);
- Press the button “Login with SPID”;
- make your choice at the opt-in screen
- Here the trick to activate test form: 5 taps inside the the banner explaining how much time you have to authenticate (see attached video for how to);
- A login form should be loaded;
- Fill the form with the provided credentials (it is case sensitive);
- Press confirm button.
I the login is successful and the app flow starts, then the PR works!
